### PR TITLE
EFI support for self-replacement of the system

### PR DESCRIPTION
**Pull request by SchoolGuy**
This should add EFI support for the case that Koan tries to self-replace.

### CODE REVIEW

**Comment by SchoolGuy** [comment]
I will mark this as a draft since I havn't tested this yet.

**Comment by SchoolGuy** [comment]
So testcases do work on my local system but not independent of the environment, I need to create some mocks to make that happen.

**Comment by SchoolGuy** [comment]
So I was able to check that the BIOS part works. UEFI is still to be tested.

**Comment by SchoolGuy** [comment]
UEFI path successfully tested. Problem is that we don't check if the kernel/initrd pair is using UEFI but that is a problem for another day.

**Review by vzhestkov — DISMISSED** [review]
Looks good

File: koan/utils.py [hunk 1]
```diff
@@ -336,7 +336,7 @@ def os_release():
     if distroname is ["debian", "ubuntu"]:
```
[the remark this answers is not in the document]
Comment (SchoolGuy) [reply koan/utils.py]: Fixed in 95b01b3

**Review by meaksh — APPROVED** [review]

**Pull request merged**